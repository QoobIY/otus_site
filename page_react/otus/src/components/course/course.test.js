import React from 'react';
import renderer from 'react-test-renderer';
import Course from '../course';

test('test Course component', () => {
    const courseComponent = renderer.create(
        <Course course={{name: 'TestCourse', description: 'Test Description', date: '21.08.1972', }}/>
    );
    const courseJSON = courseComponent.toJSON();
    expect(courseJSON).toMatchSnapshot();
});
