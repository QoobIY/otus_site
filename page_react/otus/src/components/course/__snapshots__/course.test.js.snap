// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Course component 1`] = `
<div
  className="course undefined"
>
  <div
    className="course__title"
  >
    TestCourse
  </div>
  <div
    className="course__description"
  >
    Test Description
  </div>
  <div
    className="course__date"
  >
    21.08.1972
  </div>
</div>
`;
